### ISSUE TITLE
paper.install (window) crashes Google Map

### ISSUE THREAD

**Issue by SkepticaLee** [paperjs/paper.js#1639]
On a site with a google map (e.g. https://www.skeptic.de/blog/kvi/?p=1533) paper.install (window) causes the map to freeze.

Step to reproduce:
1) Open the above page;
2) Check that the map is working, resizes, can be dragged, loads satellite vs. terrain, etc;
3) Type "paper.install (window)" into the console;
4) Map is now unresponsive, except for the placemarks.

Under some conditions (renaming the map container "gmap" and adjusting the Google map scripts accordingly), an error in paper.js line 5475 is triggered,

**Comment by sasensi** [comment]
Hi,

What `paper.install (window)` does is exposing `Paper.js` classes in the global scope.
For example, this is why you can instantiate a path by using `new Path()` instead of `new paper.Path()` after that.
So in your case, I guess that there should be a conflict in the global scope between one of `Paper.js` classes and another one used by your map library.
So the easiest solution for you is to use `Paper.js` without exposing its classes to the global scope (without calling `paper.install(window)`).
Basically, you should prefix `Paper.js` classes and variables names with `paper.`.

**Comment by lehni** [comment]
Closing since the advise given above is correct and there's nothing else to do.
